src/Stratis.Bitcoin.Tests/Consensus/ConsensusManagerTests.cs

```diff
@@ -348,8 +348,10 @@ public void BlockDownloaded_NullBlock_CallsCallbacks()
             builder.TestConsensusManager.SetupCallbackByBlocksRequestedHash(additionalHeaders.Previous.HashBlock, callback3);
 
             // call for both blocks.
-            builder.blockPullerBlockDownloadCallback(additionalHeaders.Previous.HashBlock, null, peer.Object.Connection.Id);
-            builder.blockPullerBlockDownloadCallback(additionalHeaders.HashBlock, null, peer.Object.Connection.Id);
+            var block = new Block();
+            block.ToBytes();
```

Comment:
What is the purpose of this?

Reply:
It sets the size of the block I believe?

Reply:
ah I see. The test fails otherwise.

Reply:
>It sets the size of the block I believe?

Yes. CHT uses block size which is null by default and serialization sets it